client/python/examples/cancelling.py

```diff
@@ -103,9 +126,9 @@ def quick_create_queue(client, queue):
 def workflow():
     """
     Starts a workflow, which includes:
-        - Creating a queue
-        - Creating a jobset
-        - Creating a job
+        - Creating a queue a job
```

Comment:
missing an and? Or something, this doesn't make sense as written

Reply:
Yeah this needs fixed. On it shortly